docs/src/main/asciidoc/logging.adoc

```diff
@@ -68,20 +112,27 @@ class MyService { // <2>
     }
 }
 ----
-<1> The `io.quarkus.logging.Log` class mirrors the JBoss Logging API, except all methods are `static`.
-<2> Note that the class doesn't declare a logger field.
-This is because during application build, a `private static final org.jboss.logging.Logger` field is created automatically, in each class that uses the `Log` API.
+<1> The `io.quarkus.logging.Log` class contains the same methods as JBoss Logging, except that they are `static`.
+<2> Note that the class does not declare a logger field.
+This is because during application build, a `private static final org.jboss.logging.Logger` field is created automatically in each class that uses the `Log` API.
 The fully qualified name of the class that calls the `Log` methods is used as a logger name.
 In this example, the logger name would be `com.example.MyService`.
-<3> Finally, during application build, all calls to `Log` methods are rewritten to regular JBoss Logging calls on the logger field.
+<3> Finally, all calls to `Log` methods are rewritten to regular JBoss Logging calls on the logger field during the application build.
 
 WARNING: Only use the `Log` API in application classes, not in external dependencies.
```

Comment:
@Ladicek is this still true? We changed the implementation recently if I'm not mistaken...

Reply:
The change was only for tests. Non-transformed calls to `Log` will still throw during "production" run.

Reply:
I see.